### ISSUE TITLE
Regression in v2.3.3 ignores pre-commit arguments for separation or something new?

### ISSUE THREAD

**Issue by jeking3** [asottile/reorder_python_imports#136]
I have this stanza in some repositories for pre-commit:
```
  - repo: https://github.com/asottile/reorder_python_imports
    rev: v2.3.2
    hooks:
      - id: reorder-python-imports
        language_version: python3
        args:
          - "--py37-plus"
          - "--separate-from-import"
          - "--separate-relative"
```

When I upgrade to v2.3.3 I get a ton of changes I was not expecting from a patch update.  It looks like the separate options are now ignored, or the logic has somehow changed.  It's definitely not a point release if this is the intended behavior.  All I did was update from v2.3.2 to v2.3.3 and then run pre-commit run -a, and I get diffs like:

```
jking@pulsar:~/confab/confab-cloud$ git diff confab/...
diff --git a/... b/...
index 26bc9505..d6a5714b 100644
--- a/...
+++ b/...
@@ -5,7 +5,6 @@
 #
 from confab.cloud.error import ConfigurationError
 from confab.cloud.venue.aws.adapter import AWSAdapter
-
 from ._resolver import AwsSecurityCertificateResolver
```

**Comment by jeking3** [comment]
I actually like the line separation between python import, python from, third party import, third party from sections.  Removing it is definitely not a patch release change and should be an option.

**Comment by asottile** [comment]
chill dude, probably just a mistake in #135 and definitely not an intentional change :P

bugs happen in software

**Comment by asottile** [comment]
can you share a sample file

**Comment by jeking3** [comment]
Confirmed this was resolved in v2.3.5.
